### DISCUSSION TITLE
[Vercel Blog] Support RSS

### DISCUSSION THREAD

**Discussion by devjiwonchoi** [vercel/community#6443] [Feature Question] [answered]
### Question

Can Vercel Blog support RSS feed?

Related Issue: https://github.com/vercel/next.js/issues/63589

P.S. I've subscribed to the newsletter but haven't received one yet.

**Comment by amyegan** [comment, marked as answer]
Hey @devjiwonchoi. I believe these are the links you're looking for.

- Vercel blog feed -> https://vercel.com/atom
- Next.js blog feed -> https://nextjs.org/feed.xml

**Reply by devjiwonchoi** [reply]
Perfect, thank you!!
